pom.xml

```diff
@@ -652,6 +652,7 @@
                 <module>libraries-data-io</module>
                 <module>libraries-data</module>
                 <module>libraries-files</module>
+	        <module>libraries-http-3</module>
```

Comment:
Look for all the places that libraries-http-2 is mentioned and reorder like:
                <module>libraries-http</module>
                <module>libraries-http-2</module>
                <module>libraries-http-3</module>